basic_pipeline/6_OrderingBuffer.md

```diff
@@ -140,7 +137,7 @@ defmodule Basic.Elements.OrderingBuffer do
 end
 ```
 
-We need to distinguish between two situations: the currently processed packet can have a sequence id which is subsequent to the sequence id of the last processed packet (we keep that sequence id in the state of the element) or there might be some packets not yet delivered to us, with sequence ids in between the last processed sequence id and the sequence id of a currently processed packet. In case the second situation occurs, there is nothing more we can do.
+We need to distinguish between two situations: the currently processed packet can have a sequence id which is subsequent to the sequence id of the last processed packet or there might be some packets not yet delivered to us, with sequence ids in between the last processed sequence id and the sequence id of a currently processed packet. In case the second situation occurs, there is nothing more we can do.
```

Comment:
```suggestion
We need to distinguish between two situations: the currently processed packet can have a sequence id which is subsequent to the sequence id of the last processed packet or there might be some packets not yet delivered to us, with sequence ids in between the last processed sequence id and the sequence id of a currently processed packet. In case the second situation occurs, we should store the packet and wait for the next packets to arrive.
```